mods/music/depends.txt

```diff
@@ -1 +0,0 @@
-soundset?
```

Comment:
Why remove this ? (we have a volume parameter in inventory_plus mod we added, if i remember well)

Reply:
Didn't see a mod.conf

Reply:
`mod.conf` wasn't deleted; it's still there, check it out at the `update_3d_armor` branch.